api.md

```diff
@@ -1829,22 +1809,20 @@ Update the continuous aggregate view immediately.
 REFRESH MATERIALIZED VIEW contagg_view;
 ```
 
->:TIP: Note that max_interval_per_job and refresh_lag parameter settings are used by the materialization job
-when the REFRESH is run. So the materialization (of the continuous aggregate) does not necessarily include
-all the updates to the hypertable.
-
 ---
 
-## DROP VIEW (Continuous Aggregate) :community_function: [](continuous_aggregate-drop_view)
-Continuous aggregate views can be dropped using `DROP VIEW` statement.
+## DROP MATERIALIZED VIEW (Continuous Aggregate) :community_function: [](continuous_aggregate-drop_view)
+Continuous aggregate views can be dropped using the `DROP MATERIALIZED VIEW` statement.
 
-This deletes the hypertable that stores the materialized data for the
-continuous aggregate; it does not affect the data in the underlying hypertable
-from which the continuous aggregate is derived (i.e., the raw data).  The
-`CASCADE` parameter is required for this command.
+This statement deletes the continuous aggregate and all its internal
+objects. To also delete other dependent objects&mdash;such as a view
```

Comment:
Does `&mdash;` work in the markdown parser we are using?

Reply:
I don't know for sure, but this is what the reference says (https://daringfireball.net/projects/markdown/syntax#html):
> For any markup that is not covered by Markdown’s syntax, you simply use HTML itself. There’s no need to preface it or delimit it to indicate that you’re switching from Markdown to HTML; you just use the tags.